src/test/java/com/connectfour/MoveValidatorTest.java

```diff
@@ -0,0 +1,49 @@
+package com.connectfour;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class MoveValidatorTest {
+
+    MoveValidator moveValidator;
+    Board board;
+
+    @BeforeEach
+    void initBoardAndValidator(){
+        moveValidator = new MoveValidator();
+        board = new Board();
+    }
```

Comment:
How come you're using the actual board?  What happened to the test board?

Reply:
I was hesitant to make it because all the functions ended up being copies of Board. The only difference is that TestBoard is smaller.

Reply:
Makes sense.

Reply:
Should I take it back out? Already caused an issue w/ the tests bc I had to fix both Board#addDisc and TestBoard#addDisc when I refactored earlier.

Reply:
I think it's fine to use the actual board when your test board is very similar to the actual board.